src/main/java/com/moodysalem/phantomjs/wrapper/helpers/StreamHelpers.java

```diff
@@ -0,0 +1,26 @@
+package com.moodysalem.phantomjs.wrapper.helpers;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+public class StreamHelpers {
```

Comment:
@moodysalem Done, currently not in use but it was refactored from the main PhantomJS class.